lib/config.yml

```diff
@@ -286,6 +292,9 @@ RSpec/ScatteredSetup:
 RSpec/VerifiedDoubles:
   Enabled: false
 
+Style/AmbiguousOperatorPrecedence:
+  Enabled: false
```

Comment:
I disagree with this one, I think it has been shown to provide value in the past.